reference/api/documents.mdx

```diff
@@ -248,6 +251,57 @@ An array of documents. Each document is represented as a JSON object.
 ]
 ```
 
+#### `_vectors`
+
+`_vectors` is a special document attribute containing an object with vector embeddings for AI-powered search.
+
+Each key of the `_vectors` object must be the name of a configured embedder and correspond to a nested object with two fields, `embeddings` and `regenerate`:
+
+```json
+[
+  {
+    "id": 452,
+    "title": "Female Trouble",
+    "overview": "Delinquent high school student Dawn Davenport runs away from home and embarks upon a life of crime.",
+    "_vectors": {
+      "default": {
+        "embeddings": [0.1, 0.2, 0.3],
+        "regenerate": false
+      },
+      "ollama": {
+        "embeddings": [0.4, 0.12, 0.6],
+        "regenerate": true
+      }
+    }
+  }
+]
+```
+
+`embeddings` is an optional field. It must be an array of numbers representing a single embedding for that document. It may also be an array of arrays of numbers representing multiple embeddings for that document. `embeddings` defaults to `null`.
+
+`regenerate` is a mandatory field. It must be a Boolean value. If `regenerate` is `true`, Meilisearch automatically generates embeddings for that document immediately and every time the document is updated. If `regenerate` is `false`, Meilisearch keeps the last value of the embeddings on document updates.
+
+You may also use an array shorthand to add embeddings to a document:
+
+```json
+"_vectors": {
+  "default": [0.003, 0.1, 0.75]
+}
+```
+
+Vector embeddings added with the shorthand are not replaced when Meilisearch generates new embeddings. The above example is equivalent to:
+
+```json
+"default": {
+  "embeddings": [0.1, 0.2, 0.3],
```

Comment:
We should probably use the same vector value here, otherwise it is a bit confusing 🤣 
```suggestion
  "embeddings": [0.003, 0.1, 0.75],
```